lib/ansible/modules/remote_management/redfish/redfish_command.py

```diff
@@ -277,6 +327,9 @@ def main():
             new_username=dict(aliases=["account_username"]),
             new_password=dict(aliases=["account_password"], no_log=True),
             roleid=dict(aliases=["account_roleid"]),
+            update_username=dict(type='str', aliases=["account_updatename"]),
+            account_policy_name=dict(),
+            account_policy_value=dict(),
```

Comment:
Allowing a dictionary of properties here would make a cleaner and more usable interface, allowing the user to set multiple properties at one time:

```
        account_properties=dict(type='dict', default={}),
```

This would allow plays that look something like:

```
  - name: Update account properties
    redfish_command:
      category: Accounts
      command: UpdateAccountServiceProperties
      baseuri: "{{ baseuri }}"
      username: "{{ username }}"
      password: "{{ password }}"
      account_properties:
        AccountLockoutThreshold: 5
        AccountLockoutDuration: 600
```

Reply:
Suggestion has been adopted in code. @billdodd Can you check the code again?

Reply:
Will do. Thanks for the updates.